src/commands/poll.js

```diff
@@ -55,25 +69,44 @@ exports.run = (client, message, args, callback) => {
     else if (pollOptions.length < 2) return callback("Bruder du musst schon mehr als eine Antwortmöglichkeit geben 🙄");
     else if (pollOptions.length > 10) return callback("Bitte gib nicht mehr als 10 Antwortmöglichkeiten an!");
 
-    let optionstext = `**${pollArray[0]}**\n\n`;
+    let optionstext = "";
     pollOptions.forEach((e, i) => (optionstext += `${NUMBERS[i]} - ${e}\n`));
 
     let embed = {
         "embed": {
+            "title": pollArray[0],
             "description": optionstext,
             "timestamp": moment.utc().format(),
             "author": {
                 "name": `Umfrage von ${message.author.username}`,
                 "icon_url": message.author.displayAvatarURL()
-            }
+            },
         }
     };
 
-    message.channel.send(/** @type {Object} embed */ (embed)).then(async msg => {
+    let extendable = options.extendable && pollOptions.length < 10;
+
+    if (extendable) {
+        embed["embed"]["footer"] = {
+            "text": "Umfrage ist erweiterbar mit .extend als Reply",
+        };
+
+        embed["embed"]["color"] = 'GREEN';
+    }
+
+    let channel = options.sendchannel ? client.guilds.cache.get(config.ids.guild_id).channels.cache.get(config.ids.votes_channel_id) : message.channel;
+
+    channel.send(/** @type {Object} embed */(embed)).then(async msg => {
```

Comment:
Should by explicitly typed as `@type {import("discord.js").TextChannel}` to avoid TS errors because `send` doesn't exist on VoiceChannels.